### PR TITLE
Create Tenant address ingestion function using Terraform

### PR DESCRIPTION
**Pull request by maysakanoni**

### CODE REVIEW

**Comment by maysakanoni** [comment]
[Create Tenant Azure function to read from BLOB storage container and insert address data into Cosmos DB](https://trello.com/c/U6y3Myu4/158-create-tenant-azure-function-to-read-from-blob-storage-container-and-insert-address-data-into-cosmos-db)

File: terraform/main.tf [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  resource_group_name = var.resource-group
+  location            = var.location
+
+  storage_account_name       = var.storage-account
+  storage_account_access_key = var.storage-account-primary-access-key
+  service_plan_id            = azurerm_service_plan.ingest-addresses.id
+
+  site_config {
+  }
+
```
Comment (agnieszkazbikowska-madetech) [inline terraform/main.tf]: Do we need 3 empty lines here?

**Review by agnieszkazbikowska-madetech — DISMISSED** [review]

File: terraform/main.tf (hunk 1 above)
Comment (Renny4Real) [reply terraform/main.tf]: No, we don't. Thanks for spotting 🙏🏾

Comment (Renny4Real) [reply terraform/main.tf]: Done in [8a6fd0d](https://github.com/Newarkandsherwood/HousingRepairsOnline.AddressIngestion/pull/2/commits/8a6fd0df7e2f24d7952af4e7eff0c0a33eaba047)

Comment (agnieszkazbikowska-madetech) [reply terraform/main.tf]: Thanks!

**Review by agnieszkazbikowska-madetech — DISMISSED** [review]

**Review by markWarr — DISMISSED** [review]

**Review by Renny4Real — APPROVED** [review]

**Pull request merged**